subprojects/core/src/integTest/groovy/org/gradle/caching/internal/FinalizeBuildCacheConfigurationBuildOperationIntegrationTest.groovy

```diff
@@ -190,10 +198,80 @@ class FinalizeBuildCacheConfigurationBuildOperationIntegrationTest extends Abstr
 
         then:
         def result = result()
+        result.enabled == false
+
         result.local == null
```

Comment:
@felipefzdz this doesn't seem right. Should we also be capturing the caches in this case?